typescript/ws-protocol-examples/README.md

```diff
@@ -1,22 +1,45 @@
-# Foxglove Studio WebSocket protocol examples
+# Foxglove WebSocket examples
 
-This package provides example server and client scripts using the [Foxglove Studio WebSocket protocol](https://github.com/foxglove/ws-protocol).
+This package provides example server and client implementation of the [Foxglove WebSocket protocol](https://github.com/foxglove/ws-protocol).
 
-## Usage
+For all possible actions, run the following command:
 
 ```
 $ npx @foxglove/ws-protocol-examples --help
-$ npx @foxglove/ws-protocol-examples image-server
+```
+
+## Example servers
+
+Run the `sysmon` example server, which uses JSON-encoded data to transmit system monitoring information like memory and CPU usage:
+
+```
 $ npx @foxglove/ws-protocol-examples sysmon
-$ npx @foxglove/ws-protocol-examples simple-client [host]
 ```
 
-To see data from the example servers, open up https://studio.foxglove.dev and initiate a Foxglove WebSocket connection to `ws://localhost:8765/`.
+Run the `image-server` example server, which uses JSON-encoded data to transmit images:
+
+```
+$ npx @foxglove/ws-protocol-examples image-server
+```
+
+You must close out each server (`control` + `c`) before starting up the other.
+
+To see data from either server, open [Foxglove Studio](https://studio.foxglove.dev?ds=foxglove-websocket&ds.url=ws://localhost:8765/) with a Foxglove WebSocket connection to `ws://localhost:8765/`:
+
+<img width="500" alt="Foxglove Studio displaying memory and CPU usage from the system monitor example" src="https://user-images.githubusercontent.com/14237/145313065-85c05645-6b29-4eb2-a498-849c83f8792d.png">
```

Comment:
Hm, just realized I put a screenshot of the image server in the blog post but wonder if we should do it here too... Here's one if you want it: https://user-images.githubusercontent.com/14237/146500927-4a1408c7-0725-49e7-8185-71b0280c0a8b.png